examples/js/app.js

```diff
@@ -13,6 +13,7 @@ const searchClient = algoliasearch(
 autocomplete({
   container: '#autocomplete',
   debug: true,
+  // dropdownPlacement: 'start',
```

Comment:
this was just for your testing I guess?

Reply:
It was for everybody to test the PR sandbox but it didn't deploy... We can keep it for now anyway, that's a development playground.